# Generated by Django 4.2.1 on 2023-06-07 14:22

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='FaultCommentModel',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('comment_body', models.TextField()),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('updated_at', models.DateTimeField(auto_now=True)),
            ],
            options={
                'db_table': 'fault_comment',
                'ordering': ('-created_at',),
            },
        ),
        migrations.CreateModel(
            name='FaultModel',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('title', models.CharField(max_length=250, validators=[django.core.validators.RegexValidator('^[a-zA-Z0-9А-ЯҐЄІЇа-яєії -]{2,50}$', 'only letters min 2 max 50 ch')])),
                ('body', models.TextField()),
                ('problem_solving', models.BooleanField(default=False)),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('updated_at', models.DateTimeField(auto_now=True)),
            ],
            options={
                'db_table': 'fault_msg',
                'ordering': ('-created_at',),
            },
        ),
    ]
